websites/W/Wolvesville/presence.ts

```diff
@@ -0,0 +1,271 @@
+        )
+          ? data.state = "Inventory" : false;
+
+        //Shop
+        document.querySelector(
+          "div.css-1dbjc4n.r-1awozwy.r-18u37iz.r-1w6e6rj.r-1777fci.r-1guathk"
+        )
+          ? data.state = "Shop"
+          : false;
+
+        //Settings
+        document.querySelector(
+          "div.css-1dbjc4n.r-150rngu.r-1niwhzg.r-13awgt0.r-eqz5dr.r-16y2uox.r-1wbh5a2.r-1pi2tsx.r-1lxl8vk.r-11yh6sk.r-1rnoaur.r-1sncvnh.r-13qz1uu"
+        )
+          ? data.state = "Settings" : false;
+
+        //Clan
+        const clan = document.querySelector(
+          "div.css-1dbjc4n.r-13awgt0.r-zd98yo.r-1v1z2uz.r-13qz1uu"
+        )
+          ? true : false;
+
+        if (clan) {
+          document.querySelector(
+            "div.css-1dbjc4n.r-1xfd6ze.r-13awgt0.r-1pi2tsx.r-1udh08x"
+          )
+            ? data.state = "Browsing clans..." : data.state = "Viewing their clan";
+        }
+      }
+    }
+
+    //Custom Games
+    document.querySelector(
+      "div.css-1dbjc4n.r-18u37iz.r-1777fci.r-p1pxzi.r-1u936jj.r-1a6n0ax"
+    )
+      ? data.details = "Browsing custom games..." : false;
+
+    //Lobby
+    const lobby = document.querySelector(
+      "div.css-1dbjc4n.r-13awgt0.r-gy4na3.r-wk8lta > div.css-1dbjc4n.r-1awozwy.r-led734.r-vu3uv8.r-18u37iz.r-ur6pnr.r-13qz1uu.r-136ojw6"
+    )
+      ? true
+      : document.querySelector(
+        "div.css-1dbjc4n.r-13awgt0.r-wk8lta > div.css-1dbjc4n.r-1awozwy.r-led734.r-vu3uv8.r-18u37iz.r-ur6pnr.r-13qz1uu.r-136ojw6"
+      )
+        ? true : false;
+
+    if (lobby) {
+      document.querySelector(
+        "div.css-1dbjc4n.r-1awozwy.r-173mn98.r-18u37iz"
+      )
+        ? data.details = "In a friends lobby" : data.details = "In a custom lobby";
+
+      if (showPlayerCount) {
+        const playerCountLobby = document.querySelectorAll(
+          "div.css-1dbjc4n.r-1awozwy.r-1777fci.r-1rngwi6"
+        )?.length;
+
+        if (playerCountLobby === 1)
+          data.state = `${playerCountLobby} player connected`;
+        else
+          data.state = `${playerCountLobby} players connected`;
+
+      }
+    }
+
+    //In game
+    const game = document.querySelector(
+      "div.css-1dbjc4n.r-1xfd6ze.r-d045u9.r-13awgt0.r-edyy15"
+    )
+      ? true : false;
+
+    if (game) {
+      //Pre-game Lobby
+      let preGameLobby = document.querySelector(
+        "div.css-1dbjc4n.r-1j16mh1.r-1d6rzhh.r-1loqt21.r-sga3zk.r-1sbahrg.r-1otgn73.r-lrvibr.r-7a29px > div.css-1dbjc4n.r-1awozwy.r-1pi2tsx.r-1777fci.r-13qz1uu > div.css-901oao"
+      )
+        ? true : false;
+
+      if (!preGameLobby) {
+        preGameLobby = document.querySelector(
+          "div.css-1dbjc4n.r-1j16mh1.r-1d6rzhh.r-sga3zk.r-12c3ph5.r-1sbahrg.r-lrvibr.r-7a29px > div.css-1dbjc4n.r-1awozwy.r-1pi2tsx.r-1777fci.r-13qz1uu > div.css-901oao.r-1281ybr"
+        )
+          ? true : false;
+
+        var lobbyChar;
+        if (preGameLobby) {
+          lobbyChar = document.querySelector(
```

Comment:
```suggestion
          const lobbyChar = document.querySelector(
```

Reply:
i cant do this because then it cant read the const in line 216

Reply:
Sorry, didn't see that - use `let` instead of `var`